test/release/examples/primers/arrays.chpl

```diff
@@ -127,7 +127,11 @@ writeln("After incrementing B's elements, B is:\n", B, "\n");
 // from the ranges specified within the array type's square brackets.
 // Array ``A2`` above will have the implicit domain ``{0..4}`` to
 // represent the five values in its initializing expression.
-
+//
+// When an array is modified without being specified in the iterand inside
+// of a parallel construct (like ``forall`` in the example below), it needs
+// an explicit ``ref`` intent.
```

Comment:
I suggest dropping "without being specified in the iterand". It's TMI. Also, even though users can currently get away without `ref (B)` if B is the iterand, they should not, to me the current behavior is a bug.

Reply:
Th discussion on the iterand is to distinguish between `forall a in A` and `forall i in 1..10`, where it otherwise may not be immediately obvious why the intent is not needed in the former.

I'm also not sure I agree that its a bug

Reply:
Oh. Then it is not "without being specified in the iterand", rather it is "unless the modification is through the loop's index variable."